packages/story-editor/src/components/canvas/useInsertElement.ts

```diff
@@ -30,15 +35,13 @@ import { noop } from '../../utils/noop';
 import useFocusCanvas from './useFocusCanvas';
 import getElementProperties from './utils/getElementProperties';
 
-/**
- * @param {string} type Element type.
- * @param {!Object} props The element's properties.
- * @return {Object} The new element.
- */
-function createElementForCanvas(type, props) {
+function createElementForCanvas(type: ElementType, props: Element) {
   return createNewElement(type, getElementProperties(type, props));
 }
 
+interface ElementWithPageId extends Element {
+  pageId?: string;
+}
 function useInsertElement() {
   const { addElement, combineElements, backgroundElementId } = useStory(
```

Comment:
There's always a background element, so could we up here remove the empty option from `backgroundElementId`? Would simplify some other things further down.

Reply:
There's a background element once things have been inited. In theory, it's possible that the current page is not set.

Or what did you mean by removing the empty option? 🤔

Reply:
`backgroundElementId` has type `string | undefined` in this function, which requires some extra tests throughout.